editor/src/core/model/element-metadata-utils.ts

```diff
@@ -2100,6 +2104,38 @@ export const MetadataUtils = {
       })
     }
   },
+  getNonExpressionDescendants(
+    metadata: ElementInstanceMetadataMap,
+    pathTree: ElementPathTrees,
+    target: ElementPath,
+  ): Array<ElementInstanceMetadata> {
+    const childrenInMetadata = MetadataUtils.getChildrenOrdered(metadata, pathTree, target)
+    if (!isFeatureEnabled('Code in navigator')) {
+      return childrenInMetadata
+    }
+
+    return childrenInMetadata.flatMap((c) =>
+      getNonExpressionDescendantsInner(metadata, pathTree, c),
+    )
+  },
+}
+
+function getNonExpressionDescendantsInner(
```

Comment:
The naming of this _seems_ wrong given that it's checking `isJSExpression` and including things if that is true...

Reply:
Ah, it _skips_ the expressions and directly gets their children.